### DISCUSSION TITLE
Add "Flowbite Icons"

### DISCUSSION THREAD

**Discussion by SutuSebastian** [react-icons/react-icons#918] [New Icon Set]
Website: 
https://flowbite.com/icons/

Icons:
https://github.com/themesberg/flowbite-icons/tree/main/src

Figma:
https://www.figma.com/community/file/1253280241668899805/Flowbite-Icons

Preview:

<img width="987" alt="Screenshot 2024-03-18 at 10 59 12" src="https://github.com/react-icons/react-icons/assets/41998826/0274e785-43f5-44c0-9dd1-29f800489925">

**Comment by theRuslan** [comment]
Voting for it

**Reply by SutuSebastian** [reply]
Btw we also published our own React package:

GitHub
https://github.com/themesberg/flowbite-react-icons

Npm
https://www.npmjs.com/package/flowbite-react-icons
